explorer/mempool.go

```diff
@@ -147,11 +172,31 @@ func (exp *explorerUI) storeMempoolInfo() {
 	var totalOut float64
 	var totalSize int32
 
+	exp.NewBlockDataMtx.Lock()
+	lastBlock := exp.NewBlockData.Height
+	lastBlockHash, err := exp.blockData.GetBlockHash(lastBlock)
+	if err != nil {
+		log.Warnf("Could not get block hash for last block, mempool vote filter disabled")
+	}
+	lastBlockTime := exp.NewBlockData.BlockTime
+	exp.NewBlockDataMtx.Unlock()
+
+	txindexes := make(map[string]int)
 	for _, tx := range memtxs {
 		switch tx.Type {
 		case "Ticket":
 			tickets = append(tickets, tx)
 		case "Vote":
+			if !voteForLastBlock(lastBlockHash, tx.VoteInfo.Validation.Hash, lastBlock, tx.VoteInfo.Validation.Height) {
```

Comment:
As in the other comment, I do not think a height comparison is correct, only hash.  So, `voteForLastBlock` can be removed.

If `lastBlockHash` is empty/nil, skip all this and just do the `votes` append.